SparkPost/2web2ui
# AC-1061 -  Handling No credit card present state in UI on billing page

AC-1061 -  Handling No credit card present state in UI on billing page

### What Changed
- Allow manually billed accounts with subscription.type = 'default' to load billing summary page and add credit card
- Other paid options on billing summary page, should be hidden until account adds a credit card

### How To Test
- Create a manually billing account with subscription.type = 'default'
- Open /account/billing (should load with change plan and add IP buttons disabled and no credit card message)
- Click "Update Payment Information" and add credit card successfully

- Confirm free/test accounts can upgrade and do everything they expect to do
- Confirm manually billing accounts get the banner
